audit/src/main/resources/config/liquibase/changelog/20210414-audit-record-add-document-id-changelog.xml

```diff
@@ -19,11 +19,27 @@
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
     <changeSet author="ivozaal" id="audit-record-add-document-id">
+        <preConditions onFail="MARK_RAN">
+            <dbms type="mysql"/>
+        </preConditions>
         <addColumn tableName="audit_record">
             <column name="document_id" type="BINARY(16)"/>
         </addColumn>
     </changeSet>
+
+    <changeSet author="ivozaal" id="audit-record-add-document-id-no-mysql">
+        <preConditions onFail="MARK_RAN">
+            <dbms type="postgresql"/>
```

Comment:
This is not really a "no-mysql" changeSet, this is a "postgresql" changeSet. Not sure if the hash changes if you want to add another dbms in the future.

You should be able to do a real '!mysql' by using:
`<not><dbms type="mysql"/></not>`

Reply:
Preconditions are not used in the checksum. I'm more in favor of whitelisting rather than blacklisting, so making this specific to postgresql for now is fine. Other dbms can be added in the future